src/libpriv/rpmostree-rpm-util.cxx

```diff
@@ -91,7 +91,10 @@ rpmostree_custom_nevra (GString *buffer, const char *name, uint64_t epoch, const
   gsize original_len = buffer->len;
 
   if (flags & PKG_NEVRA_FLAGS_NAME)
-    g_string_append (buffer, name);
+    {
+      g_assert (name);
```

Comment:
This makes sense to me. Should we do the same for the other `const char*` parameters as well lower down where they're used?

Reply:
At the moment I'm just quieting what the existing analyzer finds...I feel like this code really wants to be rewritten in Rust but yeah, may look at a few more assertions.